Packs/Core/ReleaseNotes/3_0_96.md

```diff
@@ -0,0 +1,33 @@
+
+#### Playbooks
+
+##### New: A source host created uncommon remote scheduled task
+
+- New: This playbook handles "Uncommon remote scheduled task creation" alerts, which are generated on the source host that created the remote scheduled task.
+
+Playbook Stages:
+
+Analysis:
+
+- The playbook verifies whether the causality process is signed. If the process is not signed, it proceeds with remediation actions; otherwise, it continues investigating the alert.
+
+Investigation:
+
+During the alert investigation, the playbook will perform the following:
+
+- Searches for related XSIAM alerts on the endpoint that use the following MITRE techniques to identify malicious activity: T1202 - Indirect Command Execution, T1021 - Remote Services.
+- Searches for related XSIAM agent alerts on the remote endpoint, to determine if the creation of the scheduled task is part of an attack pattern.
+- Searches for suspicious command-line parameters indicating a malicious scheduled task.
+
+Remediation:
+
+- Automatically disable the malicious scheduled task on the remote host.
+- Terminate the causality process.
+- Automatically Close the alert.
```

Comment:
Is it necessary to list all this information in the Release Notes?  It is in the yml file and the Readme file.

Reply:
yes